src/main/java/com/example/demo3/dao/UserEntity.java

```diff
@@ -150,6 +151,29 @@ public void setUpdatedAt(String updated_at) {
         this.updated_at = updated_at;
     }
 
+
```

Comment:
For hibernate MUST be implemented hashCode and equals 
see https://thorben-janssen.com/ultimate-guide-to-implementing-equals-and-hashcode-with-hibernate/